.gatling/example.package.conf

```
@@ -0,0 +1,64 @@
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
+
+gatling.enterprise.package {
+  # Consistent deployment using an existing package ID
+  # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+  # id = "00000000-0000-0000-0000-000000000000"
+  name = "My package name"
+  team = "My team name" # or ID with team = "00000000-0000-0000-0000-000000000000"
+  default {
+    simulation {
+      locations = [
+        {
+          name: "Europe - Paris",
+          size: 2,
+          weight: 30
+        },
+        {
+          name: "AP Pacific - Mumbai",
+          size: 2,
+          weight: 70
+        }
+      ]
+      parameters {
+        ignoreDefaults = false
+        systemProperties {
+          "com.example.prop.1" = "default value from system prop 1"
+          "com.example.prop.2" = "default value from system prop 2"
+        }
+        environmentVariables {
+          MY_SIMULATION_ENV_VAR_1 = "default value from environment 1"
+          MY_SIMULATION_ENV_VAR_2 = "default value from environment 2"
+        }
+      }
+      timeWindow {
+        rampUp = 10
+        rampDown = 10
+      }
```

A comment for the unit?
```suggestion
      timeWindow {
        rampUp = 10 # in seconds
        rampDown = 10 # in seconds
      }
```